src/CMakeLists.txt

```diff
@@ -585,6 +585,8 @@ set(BASE_SOURCES
     ui/screen/screen_welcome.h
     ui/studio.cpp
     ui/studio.h
+    ui/particlesGenerator.cpp
+    ui/particlesGenerator.h
```

Comment:
I think it supposed be in alphabetical order.

Reply:
Ah, you may be right. I haven't noticed that.

Reply:
I just also noticed that the file names don't match the style of other files. I will make a quick fix shortly. I wish the linter would catch such things too *sigh*.